.gitlab/new-e2e_testing/debian.yml

```diff
@@ -3,87 +3,132 @@
   variables:
     E2E_PLATFORM: debian
 
-.new-e2e_debian_a7_x64:
+.new-e2e_install_script:
   variables:
+    TARGETS: ./tests/agent-platform/install-script
+    TEAM: agent-platform
+    EXTRA_PARAMS: --osversion $E2E_OSVERS --platform $E2E_PLATFORM --cws-supported-osversion $E2E_CWS_SUPPORTED_OSVERS --major-version $AGENT_MAJOR_VERSION --arch $E2E_ARCH --flavor $FLAVOR
+
+.new-e2e_debian_a6_x86_64:
+  variables:
+    E2E_ARCH: x86_64
+    E2E_OSVERS: "debian-9,debian-10,debian-11,debian-12"
+    E2E_CWS_SUPPORTED_OSVERS: "debian-10,debian-11"
+    E2E_BRANCH_OSVERS: "debian-11"
+  needs: ["deploy_deb_testing-a6_x64"]
+
+.new-e2e_debian_a6_arm64:
+  variables:
+    E2E_ARCH: arm64
+    E2E_OSVERS: "debian-10"
+    E2E_CWS_SUPPORTED_OSVERS: "debian-10"
+    E2E_BRANCH_OSVERS: "debian-10"
+  needs: ["deploy_deb_testing-a6_arm64"]
+
+.new-e2e_debian_a7_x86_64:
+  variables:
+    E2E_ARCH: x86_64
     E2E_OSVERS: "debian-9,debian-10,debian-11,debian-12"
     E2E_CWS_SUPPORTED_OSVERS: "debian-10,debian-11"
     E2E_BRANCH_OSVERS: "debian-11"
   needs: ["deploy_deb_testing-a7_x64"]
 
 .new-e2e_debian_a7_arm64:
   variables:
+    E2E_ARCH: arm64
     E2E_OSVERS: "debian-10"
     E2E_CWS_SUPPORTED_OSVERS: "debian-10"
     E2E_BRANCH_OSVERS: "debian-10"
   needs: ["deploy_deb_testing-a7_arm64"]
 
+new-e2e-agent-platform-install-script-debian-a6-x86_64:
+  stage: kitchen_testing
+  extends: 
+    - .new_e2e_template
+    - .new-e2e_install_script
+    - .new-e2e_os_debian
+    - .new-e2e_debian_a6_x86_64
+    - .new-e2e_agent_a6
+  variables:
+    FLAVOR: datadog-agent
+
+new-e2e-agent-platform-install-script-debian-a6-arm64:
+  stage: kitchen_testing
+  extends: 
+    - .new_e2e_template
+    - .new-e2e_install_script
+    - .new-e2e_os_debian
+    - .new-e2e_debian_a6_arm64
+    - .new-e2e_agent_a6
+  variables:
+    FLAVOR: datadog-agent
+
 new-e2e-agent-platform-install-script-debian-a7-x64:
```

Comment:
you can maybe take the opportunity to set arch as `x86_64` here also, wdyt?

Reply:
Good point, did not even realize the naming was different